assets/css/frontend.scss

```diff
@@ -370,6 +370,111 @@ a.sensei-certificate-link {
   }
 }
 
+.sensei-quiz-pagination {
+	display: flex;
+	flex-flow: column wrap;
+	align-items: center;
+	gap: 1.5em;
+	margin: 1.618em 0;
+
+	@media only screen and (min-width: 768px) {
```

Comment:
Maybe we could add a variable for 768px and use that throughout instead?

Reply:
Done in https://github.com/Automattic/sensei/pull/4502/commits/8c0e4cadd7980e8d60b3b8d64a7d893c8e2ed9cc.